File: back/coordinates/EmptyQuadrant.php
<?php
class EmptyQuadrant extends AbstractQuadrant {
    public function __construct(int $x_sign, int $y_sign) {
        parent::__construct($x_sign, $y_sign, 'empty');
    }
    public function calcHit(float $x, float $y, float $r): bool {
        return false;
    }
}

File: src/main/java/draen/domain/quadrants/EmptyQuadrant.java
package draen.domain.quadrants;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;

public class EmptyQuadrant extends AbstractQuadrant {
    public static final String NAME = "empty";
    @JsonCreator
    public EmptyQuadrant(@JsonProperty("xMul") double xMul, @JsonProperty("yMul") double yMul) {
        super(xMul, yMul);
    }

    @Override
    protected boolean calcHit(double x, double y, double r) {
        return false;
    }

    @Override
    public String getName() {
        return NAME;
    }
}